app/src/main/res/layout/fragment_contacts.xml

```diff
@@ -11,14 +11,32 @@
         android:layout_height="wrap_content"
         app:title="@string/title_contacts" />
 
-    <com.yterletskyi.happyfriend.common.view.SearchView
-        android:id="@+id/etSearch"
+    <androidx.constraintlayout.widget.ConstraintLayout
         android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:layout_marginHorizontal="8dp"
-        android:layout_marginVertical="8dp"
-        android:hint="@string/search_hint_contacts"
-        android:inputType="text" />
+        android:layout_height="wrap_content">
+
+        <com.yterletskyi.happyfriend.common.view.SearchView
+            android:id="@+id/etSearch"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginHorizontal="8dp"
+            android:layout_marginVertical="8dp"
+            android:layout_marginEnd="5dp"
+            android:hint="@string/search_hint_contacts"
+            android:inputType="text"
+            app:layout_constraintEnd_toStartOf="@id/btnClear"
+            app:layout_constraintTop_toTopOf="parent" />
+
+        <ImageButton
+            android:id="@+id/btnClear"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:background="@drawable/ic_baseline_close_24"
+            android:visibility="gone"
+            app:layout_constraintBottom_toBottomOf="@id/etSearch"
+            app:layout_constraintEnd_toEndOf="@id/etSearch"
+            app:layout_constraintTop_toTopOf="@id/etSearch" />
```

Comment:
hmm, there appears to be no margin at the end. Can we add it please to be the same as margin top and bottom?
![image](https://user-images.githubusercontent.com/11182351/185350972-0041aa25-be3f-48fe-abc6-5106db94753b.png)